pkg/util/docker/docker_util.go

```diff
@@ -253,7 +253,7 @@ func (d *DockerUtil) InspectSelf() (types.ContainerJSON, error) {
 }
 
 // GetAgentCID returns the container ID where the current agent is running
-func (d *DockerUtil) GetAgentCID() (string, error) {
+func GetAgentCID() (string, error) {
```

Comment:
I think this function can also be private.

Reply:
Unfortunately, not anymore: I’m now using that function from `func (ku *KubeUtil) IsAgentHostNetwork()` in order to have the agent able to introspect the POD it is running in.